jolly-jellyfish/src/static/themes/console.css

```diff
@@ -0,0 +1,55 @@
+/**
+ * Green on black monospace theme.
+ * Every paragraph begins with "> " and every header with ">>> ".
+ * The final paragraph ends with a blinking cursor, as well as every header.
+ * Headers are constantly deleted and rewritten, and must be written as:
+ *     <h1><span><span>Header Text</span</span></h1>
+ */
+
+body {
+  background: black;
+  color: #0f0;
+  font-family: 'Courier New', Courier, monospace;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  display: flex;
+}
+
+h1 > span > span, h2 > span > span, h3 > span > span, h4 > span > span, h5 > span > span, h6 > span > span {
+  overflow: hidden;
+  white-space: nowrap;
+  animation: typing 20s steps(100, end) infinite;
+  display: inline-block;
+  padding-left: 1ch;
+}
+  
+@keyframes typing {
+  0% {max-width: 0}
+  50% {max-width: 100ch}
+  100% {max-width: 0}
+}
+
+h1::before, h2::before, h3::before, h4::before, h5::before {
```

Comment:
you're missing `h6`

Reply:
Also, you could make this bold maybe? I'm not sure.

Reply:
Yeah I'm missing h6 in quite a few actually, I forgot about that. Most browsers make headers bold by default AFAIK.

Reply:
Added h6